### ISSUE TITLE
request for update

### ISSUE THREAD

**Issue by johnny0213** [PrashantSaikia/Triton-for-Windows#1]
Hi, Prashant,

Thanks for your tremendous work! 
However newer versions of Triton are required to work with Flash Attention 2 on Windows. Unfortunately I failed to setup a compiler environment on Windows, so that I cannot compile by myself. On the other hand I have not found anyththing useful on the Internet.

Can you please help to compile a wheel for Windows of a newer version Triton such as this one:
https://pypi.org/project/triton/#files
triton-2.2.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (167.9 MB view hashes) Uploaded Jan 10, 2024 cp310

Much appreciated!

**Comment by dgoryeo** [comment]
Hi @johnny0213 , where you able to find a workaound to run newer version of Triton on Windows?

**Comment by DarkAlchy** [comment]
I guess this dev had a car accident and died. :(
|
triton-2.0.0-cp310-cp310-win_amd64.whl   is now a bad package and causes errors.

**Comment by PrashantSaikia** [comment]
Hey guys, apologies for not checking in here often. I had built this while I was in my previous company where my work machine was a Windows laptop. But I am in a different company now, and the work machine here is a Mac, as is my personal machine. So unfortunately, since I don't have a Windows machine anymore, I won't be able to update this.

Have you checked [here](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/customization_guide/build.html#building-for-windows-10) though?

**Comment by DarkAlchy** [comment]
Yep, but I don't use docker so I never was able to compile it myself.

**Comment by niknah** [comment]
That link points to nvidia triton server.  This package is for the openai triton?
